docs/sdk/src/guides/enable_elastic_scaling_mvp.rs

```diff
@@ -0,0 +1,142 @@
+//! # Enable elastic scaling MVP for a parachain
+//!
+//! **This guide assumes full familiarity with Asynchronous Backing and its terminology, as defined
+//! in <https://wiki.polkadot.network/docs/maintain-guides-async-backing>.
+//! Furthermore, the parachain should have already been upgraded according to the guide.**
```

Comment:
You can also link to https://github.com/paritytech/polkadot-sdk/pull/4363 once it is merged. 

Moreover, I think you can also benefit a bit from the suggestiosn similar to https://github.com/paritytech/polkadot-sdk/pull/4363#issuecomment-2141171928